apps/pie-docs/src/engineers/web-components/overview.md

```diff
@@ -0,0 +1,49 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: engineers-web-components
+    order: 1
+permalink: engineers/web-components/
+---
+
+## Introduction
+
+[Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) are a set of native browser technologies that allow us to build custom components. In other words, create custom HTML elements native to browsers. 
+
+Currently, this technology is supported by all browsers, making it possible to have one central library that will work with most frontend frameworks.
+
+The long-term aim will be to migrate JET teams over to this Web Component System, so we have one single source of truth for our global PIE components.
+
+---
+
+## Component status
+
+[pie.design](/) host the documentation for our new Web Components, you can find under `Components` in the navigation bar.
```

Comment:
```suggestion
[pie.design](/) host the documentation for our new Web Components, you can find under **Components** in the navigation bar.
```

I think it would be better to make the Component world bold, since it is not a link or path to a folder